### ISSUE TITLE
Script-Migration fails for ASP.NET Core project on .NET Framework

### ISSUE THREAD

**Issue by divega** [aspnet/EntityFramework#6465]
### Steps to reproduce
1. Create a new ASP.NET Core Web Application (.NET Framework) project with individual authentication in VS
2. Create an empty migration Foo
3. Execute the following command in PMC
 ` Script-Migration -From Foo -To 0`

### The issue
I get an error like the following:

```
System.IO.DirectoryNotFoundException: Could not find a part of the path 'c:\users\divega\documents\visual studio 2015\Projects\WebApplication11\src\WebApplication11\obj\Debug\hbbtvqqz.sql'.    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) 
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
    at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost) 
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
    at Microsoft.EntityFrameworkCore.Tools.Cli.MigrationsScriptCommand.Execute(CommonOptions commonOptions, String from, String to, String output, Boolean idempotent, String context, String environment) 
   at Microsoft.EntityFrameworkCore.Tools.Cli.MigrationsScriptCommand.<>c__DisplayClass0_0.<Configure>b__0()
    at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args) 
   at Microsoft.EntityFrameworkCore.Tools.Cli.Program.Main(String[] args)
 Could not find a part of the path 'c:\users\divega\documents\visual studio 2015\Projects\WebApplication11\src\WebApplication11\obj\Debug\hbbtvqqz.sql'. 
```

### Workarounds
Manually create the obj\Debug 

### Further technical details

EF Core version: 1.0.0 and 1.0.1
Operating system: Windows 10
Visual Studio version: 2015

**Comment by divega** [comment]
We should figure out if this is already fixed for 1.1.0.

**Comment by bricelam** [comment]
It appears that this has been fixed.
